<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFlight</class>
 <widget class="QMainWindow" name="CreateFlight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_field_time">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_destiny">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Destiny</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_create">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_available">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of available seates</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cancel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_box_id_airplane">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_box_available_seats">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="time_flight">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_label_destiny">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_flight">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_origin">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fill in the fields to create a flight</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_date">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_id_airplane">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Id Airplane</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_origin">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
